src/plugins/unified_doc_viewer/public/index.tsx

```diff
@@ -26,14 +25,11 @@ export const JsonCodeEditor = withSuspense<JsonCodeEditorProps>(
   </EuiDelayRender>
 );
 
-const LazyUnifiedDocViewer = React.lazy(() => import('./components/doc_viewer'));
-export const UnifiedDocViewer = withSuspense<DocViewRenderProps>(
-  LazyUnifiedDocViewer,
-  <EuiDelayRender delay={300}>
-    <EuiSkeletonText />
-  </EuiDelayRender>
-);
-
 export { useEsDocSearch } from './hooks';
+export { UnifiedDocViewer } from './components/lazy_doc_viewer';
+export {
+  UnifiedDocViewerFlyout,
```

Comment:
I noticed the page load bundle increased `5.9KB` with this PR. Could we also lazy load `UnifiedDocViewerFlyout`?